lib/ansible/modules/apt.py

```diff
@@ -63,44 +63,43 @@
     default: 'no'
   default_release:
     description:
-      - Corresponds to the C(-t) option for I(apt) and sets pin priorities
+      - Corresponds to the C(-t) option for C(apt) and sets pin priorities.
     aliases: [ default-release ]
     type: str
   install_recommends:
     description:
-      - Corresponds to the C(--no-install-recommends) option for I(apt). V(true) installs recommended packages.  V(false) does not install
+      - Corresponds to the C(--no-install-recommends) option for C(apt). V(true) installs recommended packages. V(false) does not install
         recommended packages. By default, Ansible will use the same defaults as the operating system. Suggested packages are never installed.
     aliases: [ install-recommends ]
     type: bool
   force:
     description:
-      - 'Corresponds to the C(--force-yes) to I(apt-get) and implies O(allow_unauthenticated=yes) and O(allow_downgrade=yes)'
-      - "This option will disable checking both the packages' signatures and the certificates of the
-        web servers they are downloaded from."
-      - 'This option *is not* the equivalent of passing the C(-f) flag to I(apt-get) on the command line'
+      - 'Corresponds to the C(--force-yes) to I(apt-get) and implies O(allow_unauthenticated=yes) and O(allow_downgrade=yes).'
+      - "This option will disable checking both the packages' signatures and the certificates of the web servers they are downloaded from."
+      - 'This option *is not* the equivalent of passing the C(-f) flag to C(apt-get) on the command line.'
       - '**This is a destructive operation with the potential to destroy your system, and it should almost never be used.**
          Please also see C(man apt-get) for more information.'
     type: bool
     default: 'no'
   clean:
     description:
       - Run the equivalent of C(apt-get clean) to clear out the local repository of retrieved package files. It removes everything but
-        the lock file from /var/cache/apt/archives/ and /var/cache/apt/archives/partial/.
+        the lock file from C(/var/cache/apt/archives/) and C(/var/cache/apt/archives/partial/).
```

Comment:
these are directories, not CLI tools

Reply:
That's exactly the reason why I propose marking them like that. Personal preferences again, I suppose.

Reply:
@bcoca 
Do you want me to roll these ones back as well? I see directory references being marked with `C()` in many other collections and even in this very repository. To me it makes sense to have them rendered in monospace. If that's to be a documentation directive for Ansible docs, I reckon there's a lot of reviewing to be done here and elsewhere to comply with that.

Reply:
@bcoca just a friendly check: is this a show stopper for you? I still think it makes sense to mark directories with `C()` and that is consistent with many other modules' docs both here and in community.general, but if it becomes a contention point I am happy to compromise to get the rest of the improvements through.